cfme/generic_objects/definition/definition_views.py

```diff
@@ -155,8 +161,13 @@ class GenericObjectDefinitionEditView(GenericObjectDefinitionAddEditView):
     @property
     def is_displayed(self):
         return (
-            self.in_generic_object_definition and
-            self.title.text == 'Edit Generic Object Class'
+            self.in_generic_object_definition
+            and self.title.text
+            == "Edit{tup[0]} Generic Object {tup[1]}".format(
+                tup=(" a", "Definition '{}'".format(self.context["object"].name))
+                if self.browser.appliance.version >= "5.11"
+                else ("", "Class")
```

Comment:
Good to avoid inline statement when it took more than 3 lines. Its difficult to read logic.

Reply:
@digitronik Updated.